lib/modules/manager/conan/__fixtures__/conanfile.txt

```diff
@@ -1,16 +1,17 @@
 [requires]
 poco/1.9.4
-zlib/[~1.2.3, loose=False]  
+zlib/[~1.2.3, loose=False]
```

Comment:
what's the change here and why?

Reply:
Sorry. This was the code formatter. The trailing spaces were removed. Shall I revert this?

Reply:
no, that's fine. didn't show up on mobile app